tools/pylint-extensions/azure-pylint-guidelines-checker/pylint_guidelines_checker.py

```diff
@@ -1365,6 +1371,11 @@ def check_parameters(self, node):
                 msgid="docstring-missing-type", args=(", ".join(missing_types)), node=node, confidence=None
             )
 
+        if misformated_types:
+            self.add_message(
+                msgid="docstring-misformatted-type", args=(", ".join(misformated_types)), node=node, confidence=None
```

Comment:
if there are multiple params that use type hints in a docstring, will this raise a linting error for each one or group them together?

Reply:
it will join them together like it does for params